app/workers/chase_references.rb

```diff
@@ -40,6 +41,20 @@ def send_14_day_chaser!
     end
   end
 
+  def send_21_day_chaser!
+    references = ApplicationReference
+                   .joins(:application_form)
+                   .feedback_requested
+                   .where(application_forms: { recruitment_cycle_year: ApplicationForm.select('candidate_id').maximum(:recruitment_cycle_year) })
+                   .where(['requested_at < ?', TimeLimitConfig.second_chase_referee_by.days.before(Time.zone.now)])
+                   .where.not(id: ChaserSent.reminder_reference_nudge.select(:chased_id))
```

Comment:
I'm a bit confused by this - does it still send even if the 14 day automatic email was already sent?

Reply:
Yeah it should send after 21 days unless it has `reminder_reference_nudge`. The numbers in the tests seem to all add up too.

Reply:
Also discussed offline, and added comments.